syncr_backend/network/handle_frontend.py

```diff
@@ -495,24 +496,32 @@ def handle_frontend_request(
     :param request:
     {
     "action": string
-    "drop_name": string
+    "drop_id": string
+    "file_path": string
     }
     :param conn: socket.accept() connection
     :return: None
     """
-    if request['drop_name'] is None:
+    if request['drop_id'] is None or request['file_path'] is None:
         response = {
             'status': 'error',
             'error': ERR_INVINPUT,
         }
     else:
-        # TODO: backend logic to subscribe to drop.
-        # TODO: Test if given drop_name is valid.
-        response = {
-            'status': 'ok',
-            'result': 'success',
-            'message': 'subscribed to drop ' + request['drop_name'],
-        }
+
+        try:
+            await sync_drop(request['drop_id'], request['file_path'])
```

Comment:
This actually does the whole sync, not just queues it.  So this page will not return until the sync finishes.  The better option would be to do `asyncio.ensure_future(sync_drop(...))` for now.